src/CGI_handler.cpp

```diff
@@ -4,55 +4,65 @@
 
```

Comment:
Is this file really still here?

Reply:
I guess this one and server.cpp with a small "s" are here to doom us, this time you caught this one before it made its way into main!